Resources/Textures/Objects/Specific/Hydroponics/corn.rsi/meta.json

```
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from https://github.com/vgstation-coders/vgstation13/commit/1dbcf389b0ec6b2c51b002df5fef8dd1519f8068",
+  "copyright": "Taken from https://github.com/vgstation-coders/vgstation13/commit/1dbcf389b0ec6b2c51b002df5fef8dd1519f8068, corn sprite created by Chaoticaa (GitHub)",
```

```suggestion
  "copyright": "Taken from https://github.com/vgstation-coders/vgstation13/commit/1dbcf389b0ec6b2c51b002df5fef8dd1519f8068, produce sprite created by Chaoticaa (GitHub)",
```

Secondary note, how does `produce_big` compare to `produce`?

---

I didn't modify produce_big so I wrote it as "corn" to avoid confusion, but I have added the change now.